xunit-autolabeler-v2/ast_parser/lib/file_utils_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import os
+
+from . import file_utils
+
+TEST_DIR = os.path.join(
+    os.path.dirname(__file__),
+    'test_data'
+)
+
+
+class GetFilesTest(unittest.TestCase):
+    def test_getfiles_ignores_dotfiles(self):
+        files = file_utils._getFiles(TEST_DIR, lambda x: True)
+        
+        assert 'dotfile_tag' not in str(files)
+
+
+class GetPythonFilesTest(unittest.TestCase):
```

Comment:
Can you add a test for `TEST_DIR/appengine/sample.py`?

Reply:
Done.

Reply:
Ok, can you also add

`TEST_DIR/appengine/library.py`
and
`TEST_DIR/appengine/lib/should_be_ignored.py`
?